stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -499,13 +501,44 @@ public fun DefaultMessageItemContent(
 }
 
 /**
- * Container for all the reactions this message has.
+ * Represents a reaction bubble with a list of reactions this message has.
  *
+ * @param message The message we are displaying reactions for.
  * @param modifier Modifier for styling.
- * @param reactions Map of reactions and their count.
  */
 @Composable
-private fun MessageReactions(
+public fun MessageReactions(
+    message: Message,
+    modifier: Modifier = Modifier,
+) {
+    val ownReactions = message.ownReactions
+    val supportedReactions = ChatTheme.reactionTypes
+
+    val reactions = message.reactionCounts
+        .map { it.key }
+        .filter { supportedReactions[it] != null }
```

Comment:
Maybe we can filter for the reactions that are supported within the app and then have the `isNotEmpty()` check, after which we map them to the pair?

Also - I think reactions should be a `Pair<String, Painter>`, as that way users can load custom emoji reactions (e.g. like in Slack), as that way they can provide us with any type of Painter to load the image.

Finally - I don't really like that we have 2 components with the name `MessageReactions` - can we put this logic into one component as to me this is a bit confusing.

We should also make this customizable by exposing a lambda that represents the content of each reaction item (the icon in our case) so that our users can override what they look like (and potentially apply their own logic). 

If we can, we could also remove the default hardcoded modifiers as our users might want to have the `background` as transparent.

Can you please apply these changes?